### DISCUSSION TITLE
Moving model(s) has taken 21.13 seconds

### DISCUSSION THREAD

**Discussion by ycmomo** [lllyasviel/Fooocus#3052] [Q&A] [answered]
I found that the Moving model (s) takes a long time. Is there any solution to this problem?

**Comment by mashb1t** [comment, marked as answer]
see https://github.com/lllyasviel/Fooocus/issues/3053, this is also not necessarily a problem / issue, just not so fast hardware on your end.

**Reply by ycmomo** [reply]
Thank you very much for your answer
